### ISSUE TITLE
No error on line overflow with a string literal

### ISSUE THREAD

**Issue by YaLTeR** [rust-lang-nursery/rustfmt#2459]
This doesn't produce an error even though it doesn't fit within `max_width`:
```rust
fn main() {
    "This is a really really really really really really really really really really really really really long message";
}
```
Default settings, `0.4.0-nightly (bdf56292 2018-02-17)`

**Comment by topecongiro** [comment]
Isn't this an expected behavior? `rustfmt` should not warn on comments of string literals by default.

**Comment by topecongiro** [comment]
Closed via #2483.
